app/models/planning_application.rb

```diff
@@ -65,8 +66,16 @@ class PlanningApplication < ApplicationRecord
       transitions from: %i[not_started invalidated in_assessment], to: :in_assessment, guard: :has_validation_date?
     end
 
+    event :save_assessment do
+      transitions from: %i[in_assessment assessment_in_progress], to: :assessment_in_progress
```

Comment:
yes, this happens because we want to allow the save_assessment to be called multiple times whenever someone hits that save draft button - it didn't complain thus far